app/src/main/java/cs465/illinois/edu/dogonthequad/User.java

```diff
@@ -8,4 +8,8 @@
 
     public UUID mId = UUID.randomUUID();
 
+    public UUID getmId() {
```

Comment:
You probably want to name this `getId`, since the `m` is a marker that a variable is a member variable.